x/data/server/msg_register_resolver.go

```diff
@@ -0,0 +1,53 @@
+package server
+
+import (
+	"bytes"
+	"context"
+
+	cosmossdk "github.com/cosmos/cosmos-sdk/types"
+
+	api "github.com/regen-network/regen-ledger/api/regen/data/v1"
+	sdk "github.com/regen-network/regen-ledger/types"
```

Comment:
i think we usually just call this regentypes or just types

Reply:
We actually don't need this and can use `sdk github.com/cosmos/cosmos-sdk/types`.

Reply:
https://github.com/regen-network/regen-ledger/pull/956/commits/5fa93135c4d11c1cde2b44d349c4157587f35923